python/ray/rllib/test/test_checkpoint_restore.py

```diff
@@ -26,11 +25,14 @@ def get_mean_action(alg, obs):
 for (cls, default_config) in [
         (DQNAgent, DQN_CONFIG),
         (PPOAgent, PG_CONFIG),
-        # TODO(ekl) this fails with multiple ES instances in a process
+        # https://github.com/ray-project/ray/issues/1062
         # (ESAgent, ES_CONFIG),
-        (A3CAgent, A3C_CONFIG)]:
+        # https://github.com/ray-project/ray/issues/1061
+        # (A3CAgent, A3C_CONFIG)
+        ]:
     config = default_config.copy()
     config["num_sgd_iter"] = 5
+    config["use_lstm"] = False
```

Comment:
This is now A3C specific, this wouldn't throw an error here so maybe it's ok? also consider noting this in case of future confusion.